docs/src/index.md

```diff
@@ -1,14 +1,349 @@
+
+ds_native = MetopDataset(input_file)
+ds_nc = NCDataset(output_file, "c") 
+NCDatasets.write(ds_nc, ds_native)
+
+close(ds_native)
+close(ds_nc)
+```
+It is also possible to use the safe `do` syntax that ensures the files are closed correctly even in the case of exceptions.
+
+```julia
+using MetopDatasets
+using NCDatasets
+
+input_file = "ASCA_SZR_1B_M01_20190109125700Z_20190109143858Z_N_O_20190109134816Z.nat"
+output_file = "ASCA_SZR_1B_M01_20190109125700Z_20190109143858Z_N_O_20190109134816Z.nc"
+
+MetopDataset(input_file) do ds_native
+  NCDataset(output_file, "c") do ds_nc
+    NCDatasets.write(ds_nc, ds_native)
+  end
+end
+```
+
+## Supported formats
+- ASCAT Level 1B
+- ASCAT Level 2 Soil Moisture
+- IASI Level 1C
+
+### Formats not yet supported
+- AMSU-A Level 1B
+- AVHRR Level 1B
+- GOME-2 Level 1B
+- HIRS Level 1B 
+- IASI Level 2 Combined Sounding
+- MHS Level 1B 
+
+### Reference documents 
+- [EPS Generic Product Format Specification](https://user.eumetsat.int/s3/eup-strapi-media/pdf_gen_pfs_13e3f0feb7.pdf)
+- [ASCAT Level 1: Product Format Specification](https://user.eumetsat.int/s3/eup-strapi-media/ASCAT_Level_1_Product_Format_V12_Annexe_50fe72d349.pdf)
+- [ASCAT Level 2 Soil Moisture: Product Format Specification](https://user.eumetsat.int/s3/eup-strapi-media/pdf_ten_0343_eps_ascatl2_pfs_f509981295.pdf)
+- [IASI Level 1 Product Format Specification](https://user.eumetsat.int/s3/eup-strapi-media/pdf_iasi_level_1_pfs_2105bc9ccf.pdf)
+- [IASI Level 2 Product Format Specification](https://user.eumetsat.int/s3/eup-strapi-media/pdf_ten_980760_eps_iasi_l2_f9511c26d2.pdf)
+
+## Development status and versioning
+The package was previously named **MetopNative.jl** and was hosted on the [EUMETSAT GitLab](https://gitlab.eumetsat.int/eumetlab/cross-cutting-tools/MetopNative.jl).
+
+This package is still in the early development phase and is not yet registered. The plan is to register the package once it is complete for ASCAT and IASI products. The package will not follow any specific version system before it is registered.
+
+After registration, the aim will be to follow the [sematic versioning](https://semver.org/) system. Note that the package will start as 0.x.y to signal that breaking changes is to be expected. This is done to allow for more rapid development and because major dependencies like the [CommonDataModel.jl](https://github.com/JuliaGeo/CommonDataModel.jl) are not at the version 1.0 milestone yet. It is therefore recommended to use [Pkg environments](https://pkgdocs.julialang.org/v1/compatibility/) for projects with MetopDatasets.jl to handle comparability and ensure reproducibility. Please note that many of the "breaking changes" will be small and only affect specific use cases. This could for example be the correction of a single variable name. In breaking change will be listed in the [release description](https://github.com/eumetsat/MetopDatasets.jl/releases).
```

Comment:
sematic -> semantic
changes is -> changes are
In breaking change -> Any breaking change